src/containers/Welcome/Step4/Step4.tsx

```diff
@@ -157,7 +166,7 @@ const Step4 = (p: Wizard.StepProps) => {
             <Trans i18nKey="consent:paragraph1">
               Virufy cares about your privacy and is advised by licensed data privacy experts.
               The information and recordings you provide will only be used for the purposes described in our
-              Privacy Policy and consent form.
+              <LinkPurple to={privacyPolicy} target="_blank">Privacy Policy</LinkPurple> and consent form.
```

Comment:
Is not necessary to store privacy policy in a const, please just put the link into `to` prop, like this:
`to="https://virufy.org/privacy_policy"`